src/pages/NonstopScans/main.ts

```diff
@@ -0,0 +1,85 @@
+import { pageInterface } from '../pageInterface';
+
+export const NonstopScans: pageInterface = {
+  name: 'NonstopScans',
+  domain: 'https://www.nonstopscans.com/',
```

Comment:
The domain should always be without a trailing /

Reply:
Removed Trailing /